sdk/src/main/scala/com/horizen/account/state/MessageProcessorUtil.scala

```diff
@@ -0,0 +1,15 @@
+package com.horizen.account.state
+
+import com.horizen.params.NetworkParams
+
+import scala.collection.JavaConverters.asScalaBufferConverter
+
+object MessageProcessorUtil {
+  def getMessageProcessorSeq(params: NetworkParams, customMessageProcessors: java.util.List[MessageProcessor]): Seq[MessageProcessor] = {
+    Seq(
+      EoaMessageProcessor,
+      WithdrawalMsgProcessor,
+      ForgerStakeMsgProcessor(params),
+    ) ++ customMessageProcessors.asScala
```

Comment:
So we may do `.asScala` in bootstrapping tool class.

Reply:
ok